apps/hpc-ftsadmin/src/app/components/flow-form.tsx

```diff
@@ -0,0 +1,4094 @@
+    const newArray = [...array];
+    newArray[index] = {};
+    return newArray;
+  };
+  if (remove === true) setRemove(false);
+  const SourceLink = (
+    setFieldValue: any,
+    values: FormValues,
+    indexKey: number,
+    index: number
+  ) => {
+    setFieldValue(
+      `reportDetails[${index}].reportedOrganization`,
+      values.sourceOrganizations[indexKey]
+    );
+  };
+  const DestinationLink = (
+    setFieldValue: any,
+    values: FormValues,
+    indexKey: number,
+    index: number
+  ) => {
+    setFieldValue(
+      `reportDetails[${index}].reportedOrganization`,
+      values.destinationOrganizations[indexKey]
+    );
+  };
+
+  interface Inconsistency {
+    type: string;
+    values: {
+      name?: string;
+      year?: string;
+      refDirection: string;
+      options?: { name: string }[];
+    }[];
+  }
+
+  const processDataInconsistencies = (
+    inconsistencyArray: Inconsistency[]
+  ): string => {
+    let message = '';
+    inconsistencyArray.forEach((inconsistencyWith) => {
+      if (inconsistencyWith && inconsistencyWith.type) {
+        const inconsistencyType = inconsistencyWith.type;
+        message +=
+          inconsistencyType.charAt(0).toUpperCase() +
+          inconsistencyType.slice(1) +
+          ': ';
+
+        if (inconsistencyWith.type === 'no-direct-link') {
+          message += inconsistencyWith.values.join(', ');
+        } else {
+          message +=
+            inconsistencyWith.values
+              .map((value) => {
+                const name = value.name || value.year;
+                let joinedOptions = '';
+                if (value.options) {
+                  joinedOptions = value.options
+                    .map((option) => {
+                      return option.name || JSON.stringify(option);
+                    })
+                    .join(', ');
+                }
+                const options = `is not in the list of acceptable ${value.refDirection} ${inconsistencyType}: [${joinedOptions}]`;
+                return `'${name}' ${options}`;
+              })
+              .join(', ') + '. ';
+        }
+      } else {
+        message += JSON.stringify(inconsistencyArray);
+      }
+    });
+    return message;
+  };
+
+  const handleSubmit = async (
+    values: FormValues,
+    submitAction: string | undefined
+  ) => {
+    if (values.childFlow) {
+      for (let i = 0; i < values.childFlow.length; i++) {
+        if (
+          values.destinationUsageYears.length >= 2 &&
+          JSON.parse(values.childFlow[i].value as string).activeStatus === true
+        ) {
+          values.flowType = {
+            displayLabel: 'Parked',
+            value: '1252',
+          };
+        }
+      }
+    }
+    if (isShowParentFlow && isShowChildFlow > 0) {
+      let childAmountSum = 0;
+      let parentAmountSum = 0;
+      values.childFlow &&
+        values.childFlow.map((item, _index) => {
```

Comment:
Even if you properly added '_' before index to declare it is unused, there is no need to even write it in this case. Please go around the code checking for this and delete them if unused.
```suggestion
        values.childFlow.map((item) => {
```